src/libcalamaresui/Branding.h

```diff
@@ -67,10 +67,11 @@ class UIDLLEXPORT Branding : public QObject
 
     enum ImageEntry : short
     {
-        ProductLogo,
+        ProductBanner,
```

Comment:
I get the impression there's a corrsponding `Branding.cpp` change that is missing here.

Reply:
yes there is - i forgot to include it - i re-implemented this feature by hand onto the master branch without building it, rather than cherry-picking the buried commits